openhands/controller/agent_controller.py

```diff
@@ -532,7 +534,7 @@ async def start_delegate(self, action: AgentDelegateAction) -> None:
         agent_cls: Type[Agent] = Agent.get_cls(action.agent)
         agent_config = self.agent_configs.get(action.agent, self.agent.config)
         llm_config = self.agent_to_llm_config.get(action.agent, self.agent.llm.config)
-        llm = LLM(config=llm_config)
+        llm = LLM(config=llm_config, retry_listener=self._notify_on_llm_retry)
```

Comment:
Check out `self._status_callback`--I think that's the right way to do this

Reply:
@rbren We'll have to talk more about what you mean by that. I'm trying to do something similar to status_callback here, but what it does is actually change a status and while we're in the middle of a retry loop the status of the agent hasn't really changed.

Reply:
I do think the little "status indicator" next to the play/pause button is the right place for something like this. Not sure at what point we'd clear it out though.

We have a pattern of sending back a single space `' '` (why not the empty string? no idea) to clear the status message

Reply:
OK, should be more like you envisioned now, you can take a look when you get a chance

Reply:
Yeah this is perfect